digicampipe/tests/test_imports.py

```diff
@@ -1,13 +1,17 @@
-# This just checks if all imports from pipeline_crab.py work
+# This just checks some imports work
 
-from digicampipe.calib.camera import filter, r1, random_triggers, dl0, dl2, dl1
-from digicampipe.io.event_stream import event_stream
-from digicampipe.utils import geometry
-from cts_core.camera import Camera
-from digicampipe.io.save_hillas import save_hillas_parameters
-from digicampipe.io.save_hillas import save_hillas_parameters
-from digicampipe.visualization import EventViewer
-from digicampipe.utils import utils
-import protozfitsreader
-from protozfitsreader import rawzfitsreader
-from protozfitsreader import ZFile
+
+def test_imports():
+
+    from digicampipe.calib.camera import filter, r1, random_triggers, dl0, dl2, dl1
+    from digicampipe.io.event_stream import event_stream
+    from digicampipe.utils import geometry
+    from cts_core.camera import Camera
+    from digicampipe.io.save_hillas import save_hillas_parameters
+    from digicampipe.io.save_hillas import save_hillas_parameters
+    from digicampipe.visualization import EventViewer
+    from digicampipe.utils import utils
+    import protozfitsreader
+    from protozfitsreader import rawzfitsreader
+    from protozfitsreader import ZFile
+    from histogram.histogram import Histogram1D
```

Comment:
This `test_imports` is from the very very early days, where I got immediately a lot of Exceptions in my fact, when I only wanted to try any example. And the exceptions occured already at the import lines, that is in the very first lines of the script...

Since there were no unit tests what so ever, I thought ... we at least test if the imports work ... at least this.

however this does not mean that we should maintain this test and make sure new stuff is imported here. What we should do is: have at least one unit test per whatever we import.

So instead of checking if `from histogram.histogram import Histogram1D` really works ... why not write a single little unit test which shows in which context Histogram+D is used ... and so on.

Ultimately we have to get rid of this test here and make sure we have real tests ...

Reply:
I just made a test form the container to histogram func. so I can remove it. I will remove the imports that don't need to be testes e.g. protozfitsreader (some tests already perfomed in protozfitsreader package)